### PR TITLE
Feature: hide unsafe images & performance

### PR DESCRIPTION
**Pull request by YegorZaremba**
Fixed: #9 and #10 

Implementation of "hide unsafe images" logic
Draft status 'cause I would like to dive into `background.js` and improve errors a little bit

### CODE REVIEW

**Comment by navendu-pottekkat** [comment]
I will test out this PR and merge by EOD

**Comment by YegorZaremba** [comment]
Thanks for your response, I just found a bug with lazy loading, I will fix it in 7 hours
Ping you when be fixed
And errors improvment in `backround.js` will be in new ticket/PR

**Comment by navendu-pottekkat** [comment]
Sure man! I was caught up in the middle of some work. I will just test this PR out and then merge just to make sure. You can make the other PR also. Great work!

**Comment by YegorZaremba** [comment]
@navendu-pottekkat I finished with lazy loading and delay of js rendering
Tested in firefox and chrome, looks fine for me

File: src/content.js [hunk 1]
```diff
@@ -6,85 +6,71 @@
...
+
+        // lazy load handler
+        const lazyLoadKeys = ['lazy', 'load']
+        const isImageLazyLoad = lazyLoadKeys.find(key => JSON.stringify([images[i].src, images[i].classList, images[i].className]).includes(key))
+        if (!isImageLazyLoad) {
+          images[i].__isChecked = true
+        }
+      }
+
+      // @todo handle unsafe images smaller than 64px
```
Comment (navendu-pottekkat) [inline src/content.js:41]: This also is a good idea.

**Comment by navendu-pottekkat** [comment]
@YegorZaremba Very awesome work. I will make a contributors section and add you

**Comment by navendu-pottekkat** [comment]
@all-contributors please add YegorZaremba for code, design, ideas

**Comment by YegorZaremba** [comment]
Thanks @navendu-pottekkat, I'll do some improvements in the next week, if you planing to release new beta version, please move date to end of the week
Holy molly amazing `allcontributors` bot :smile: 
ps. remove `anonacc`, its my fake git account :smile: (yeah I'm busted)

**Comment by navendu-pottekkat** [comment]
@YegorZaremba I published a beta anyway! These are very good improvements from before, so I have merged it with master. We can go for a proper release next week after you have made your changes. I have not had the chance to extensively test it though but I think you have tested it enough?

**Comment by YegorZaremba** [comment]
Yeah, if compare with previous version it's better
I have tested enough, but there are some bugs in specific cases, I will try make production quality in the next week

**Comment by navendu-pottekkat** [comment]
As long as the extension works now

**Pull request merged**